$NetBSD$

Disable pulse; alt. fix pulseaudio.h

 -void write_audio (void *bla, int *in, size_t size);
 +void write_audio (void *bla, int *in, int size);


--- Makefile.orig	2013-01-06 14:14:09.000000000 +0000
+++ Makefile
@@ -8,7 +8,7 @@ DESTDIR?=/usr
 USE_CA=NO
 
 # set to YES if you want to use PulseAudio instead of OSS
-USE_PA=YES
+USE_PA=NO
 
 # set to YES if you compile with MINGW32
 USE_WIN32=NO
@@ -19,7 +19,7 @@ OSX_PLATFORM=NO
 # set to YES if you want make install to build an OSX bundle instead of
 # installing to DESTDIR
 # also directs qrq to look in bundle location for shared resources
-OSX_BUNDLE=YES
+OSX_BUNDLE=NO
 
 ifneq ($(OSX_PLATFORM), YES)
 		OSX_BUNDLE=NO
